cmd/cluster/validatepullsecret.go

```diff
@@ -94,6 +95,9 @@ func (o *validatePullSecretOptions) run() error {
 
 func (o *validatePullSecretOptions) getPullSecretFromCluster() (email string, err error, sentSL bool) {
 	if o.elevate {
+		if len(o.reason) == 0 {
```

Comment:
Instead of doing this check here, I believe we could just add `validatePullSecretCmd.MarkFlagsRequiredTogether(...)` after line 54. That way we let Cobra handle the logic and keep our code cleaner. Thoughts?

Reply:
Great, that was what I was looking for, but did not find it! Will change that ;) Thanks...
/hold

Reply:
Implemented as you suggested.
The only drawback is that if we specify --elevate=false, then --reason will also be mandatory, but I guess we will ever do that, and it is not really an issue...